config/config.go

```diff
@@ -663,6 +667,12 @@ type QueueConfig struct {
 	// On recoverable errors, backoff exponentially.
 	MinBackoff model.Duration `yaml:"min_backoff,omitempty"`
 	MaxBackoff model.Duration `yaml:"max_backoff,omitempty"`
+
+	// Metadata controls whenever we send metric metadata to remote storage.
+	Metadata bool `yaml:"metadata"`
```

Comment:
nit; these names, or at least the yaml, might make more sense as `SendMetadata/send_metadata` and `MetadataSendInterval/metadata_send_interval`

Reply:
Makes sense!